workflow/exp/local.conus12km

```diff
@@ -25,12 +25,12 @@ case ${MACHINE} in
     export QUEUE="batch"
     export PARTITION="kjet"
     ;;
-  "gaeac6")
+  "gaea")
```

Comment:
As Matt mentioned, this file no longer exists.
You can add the change on `QUEUE` to the `exp.conus12km` file.